docs/codeql/support/reusables/frameworks.rst

```diff
@@ -28,6 +28,7 @@ C# built-in support
    Json.NET, Serialization
    NHibernate, Database ORM
    WinForms, User interface
+   Dapper, Database ORM
```

Comment:
Minor comment: Could you move this up above "EntityFramework", so the list stays in alphabetical order? 🙂